<launch>
	<arg name="rviz" default="true" />
	<arg name="image" default="false" />
	<arg name="calib" default="false" />
	<arg name="calib_in" default="calibration_initial" />
	<arg name="calib_out" default="new_calibration" />
	<arg name="calib_dir" default="$(find elikos_detection)/cfg/" />

  <node name="elikos_detection" pkg="elikos_detection" type="elikos_detection">
      <param name="calibration" value="$(arg calib)" />
      <param name="file_in" value="$(arg calib_in)" />
      <param name="file_out" value="$(arg calib_out)" />
      <param name="dir" value="$(arg calib_dir)" />
  </node>

  <node name="elikos_transformation" pkg="elikos_detection" type="elikos_transformation" />
</launch>
